### DISCUSSION TITLE
Issue with Git

### DISCUSSION THREAD

**Discussion by onurakinsir** [cafawo/FinancialDataAnalytics#14] [Q&A]
I am having an issue with Git. I would like to pull (update the repository from https://github.com/cafawo/FinancialDataAnalytics) but when I type the code on the terminal:

```
cd /Users/OA/Desktop/documents/FinancialDataAnalytics
git pull https://github.com/cafawo/FinancialDataAnalytics
```

It states that it is already up to date. But it does not update the repository. 

P.S. I first cloned this repository onto my Downloads folder and I changed its directory.

**Comment by OlgaKuzminichna** [comment]
if you type git remote -v, what it shows?
I had the same problem and it appeared that I was connected to a cloned version, which I was not updating.
I would either update cloned version in the website or connect directly to the main folder

**Reply by onurakinsir** [reply]
When I type:
- `git remote -v`

It does not provide either an error or information. However, when I try to clone it again, it states that:

> fatal: destination path 'FinancialDataAnalytics' already exists and is not an empty directory.

**Reply by OlgaKuzminichna** [reply]
Try first open a folder, where you cloned
then  click git bush 

![image](https://github.com/cafawo/FinancialDataAnalytics/assets/120052171/c7c0d4c2-8890-4c99-a83d-54e2babd6803)

with this way git remote will show where you connected

![image](https://github.com/cafawo/FinancialDataAnalytics/assets/120052171/bd9641a7-f320-4cc1-8fa8-1bb364788052)

If you want to clone again then delete the previous folder or locate in totally different path

**Reply by onurakinsir** [reply]
I am afraid it still does not work. When I try to update the repository – it returns as "_Already up-to-date_" but it does not at all! But thanks for your effort anyway!

**Reply by cafawo** [reply]
Dear @onurakinsir, please replicate the screenshot that @olyafrontend provided using `git remote -v` and upload it here. Otherwise it is hard to track down your error. 

If that is not fruitful, `git clone https://github.com/cafawo/FinancialDataAnalytics.git` into a different directory and try again.
